src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/StopTimeTestsIT.java

```diff
@@ -0,0 +1,142 @@
+      .body("arrivalTime[1]", is(getSecondsFromMidnight("15:06:00")))
+      .body("departureTime[1]", is(getSecondsFromMidnight("15:06:00")))
+      .body("sequency[1]", is(2));
+    
+    String tripId = response.body().jsonPath().getString("tripId[1]");
+    String stopId = response.body().jsonPath().getString("stopId[1]");
+    
+    assertFound(String.format("/organizations/%s/transportTrips/%s", organizationId, tripId));
+    assertFound(String.format("/organizations/%s/transportStops/%s", organizationId, stopId));
+  }
+  
+  @Test
+  public void testFindStopTime() {
+    String organizationId = getOrganizationId(0);
+    
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportStopTimes/{stopTimeId}", organizationId, getOrganizationStopTimeId(organizationId, 2));
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("tripId", notNullValue())
+      .body("stopId", notNullValue())
+      .body("arrivalTime", is(getSecondsFromMidnight("15:07:00")))
+      .body("departureTime", is(getSecondsFromMidnight("15:07:00")))
+      .body("sequency", is(3));
+    
+    String tripId = response.body().jsonPath().getString("tripId");
+    String stopId = response.body().jsonPath().getString("stopId");
+    
+    assertFound(String.format("/organizations/%s/transportTrips/%s", organizationId, tripId));
+    assertFound(String.format("/organizations/%s/transportStops/%s", organizationId, stopId));
+  }
+  
+  @Test
+  public void testOrganizationStopTimesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationStopTimeId = getOrganizationStopTimeId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportStopTimes/%s", organizationId, organizationStopTimeId));
+    assertEquals(19, countApiList(String.format("/organizations/%s/transportStopTimes", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/transportStopTimes/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/transportStopTimes/%s", incorrectOrganizationId, organizationStopTimeId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/transportStopTimes", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+  private void createGtfsSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createGtfsSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)